### PR TITLE
fix(vehicle): Make VIN number generation more accurate

### PR DESCRIPTION
**Pull request by lulunac27a**
The last six digits of VIN (vehicle identification number) is the number and the 13th digit (last five digits) can start with number 0 (zero).

### CODE REVIEW

File: src/modules/vehicle/index.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class VehicleModule extends ModuleBase {
       length: 1,
       casing: 'upper',
       exclude,
-    })}${this.faker.number.int({ min: 10000, max: 99999 })}`; // return five digit #
+    })}${this.faker.number.int({ min: 0, max: 99999 }).toString().padStart(5, '0'}`; // return five digit # with leading zeros
```
Comment (matthewmayer) [inline src/modules/vehicle/index.ts]: Probably better to use `faker.string.numeric` for this.

**Review by matthewmayer — DISMISSED** [review]

File: src/modules/vehicle/index.ts (hunk 1 above)
Comment (matthewmayer) [reply src/modules/vehicle/index.ts]: Also note there's a typo in the code the last } should be )

File: src/modules/vehicle/index.ts [hunk 2]
```diff
@@ -96,7 +96,7 @@ export class VehicleModule extends ModuleBase {
       length: 1,
       casing: 'upper',
       exclude,
-    })}${this.faker.number.int({ min: 10000, max: 99999 })}`; // return five digit #
+    })}${this.faker.number.int({ min: 0, max: 99999 }).toString().padStart(5, '0')}`; // return five digit # with leading zeros
```
Comment (xDivisionByZerox) [inline src/modules/vehicle/index.ts]: By using a faker method, the function becomes more readable IMO.

```suggestion
    })}${this.faker.string.numeric({ length: 5, allowLeadingZeros: true })}`;
```

**Comment by xDivisionByZerox** [comment]
@faker-js/maintainers Is this a `refactor` or a `fix`?

**Comment by ST-DDT** [comment]
> @faker-js/maintainers Is this a `refactor` or a `fix`?

IMO `fix`

**Comment by Shinigami92** [comment]
FFR: https://en.m.wikipedia.org/wiki/Vehicle_identification_number

**Review by xDivisionByZerox — DISMISSED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

File: src/modules/vehicle/index.ts (hunk 1 above)
Comment (RD-CL) [reply src/modules/vehicle/index.ts]: Matt Mayer,

You have not responded to emails and phone calls. RDCL will provide all of your personal information and evidence to the police investigators and the relevant authorities. Your representation in Hong Kong say they have no instructions to act in this matter. Your actions are criminal and this is theft. RDCL will be relentless in the pursuit of its property and you will be notified daily through all online channels until you return what you have stolen and provide all logins and passwords associated with the accounts and turn over all information on to whom you have been illegally redirecting RDCL's confidential email, including financial data.

RDCL will communicate with you in open forum to ensure you receive these messages.

**Pull request merged**